test/csfle-kms-providers.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// TODO(NODE-6685): use secrets manager variables
+// const csfleKMSProviders = {
+//     "aws": {
+//         "accessKeyId": process.env.FLE_AWS_KEY,
+//         "secretAccessKey": process.env.FLE_AWS_SECRET
+//     },
+//     "azure": {
+//         "tenantId": process.env.FLE_AZURE_TENANTID,
+//         "clientId": process.env.FLE_AZURE_CLIENTID,
+//         "clientSecret": process.env.FLE_AZURE_CLIENTSECRET
+//     },
+//     "gcp": {
+//         "email": process.env.FLE_GCP_EMAIL,
+//         "privateKey": process.env.FLE_GCP_PRIVATEKEY
+//     },
+//     "local": {
+//         "key": Buffer.from("Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk", 'base64')
+//     },
+//     "kmip": {
+//         "endpoint": "localhost:5698"
+//     }
+// }
+
+function getCSFLEKMSProviders() {
+  return JSON.parse(process.env.CSFLE_KMS_PROVIDERS2 ?? '{}');
+}
+
+const keys = [
+  'FLE_AWS_KEY',
+  'FLE_AWS_SECRET',
+  'FLE_AZURE_TENANTID',
+  'FLE_AZURE_CLIENTID',
+  'FLE_AZURE_CLIENTSECRET',
+  'FLE_GCP_EMAIL',
+  'FLE_GCP_PRIVATEKEY'
+];
+
+module.exports = {
+  getCSFLEKMSProviders,
```

Comment:
I'm happy to remove this if anybody has a better suggestion.

We have logic in our test suite that skips tests depending on whether the KMS credentials are set (CSFLE_KMS_PROVIDERS is in the environment).  That variable won't be there going forward, so I had to figure something else out.  

This list of variables corresponds to all the secrets contained in CSFLE_KMS_PROVIDERS.  If any are unset, I skip tests in the same way that we used to skip if CSFLE_KMS_PROVIDERS is unset.